server/ColorDeconvolution/ColorDeconvolution.xml

```diff
@@ -60,5 +60,42 @@
       <longflag>stainColor_3</longflag>
       <default>0,0,0</default>
     </double-vector>
+    <boolean>
+      <name>largeImage</name>
+      <label>Large Image?</label>
+      <longflag>largeImage</longflag>
+      <description>True if the input image is a large image that should be handled with large_image</description>
+    </boolean>
```

Comment:
No need for this bool parameter. You can use [this](you can use [this](https://github.com/DigitalSlideArchive/HistomicsTK/blob/master/server/NucleiDetection/NucleiDetection.py#L225) for testing if the image is a whole-slide image) for testing if the image is a whole-slide image.